src/member/constants.test.ts

```diff
@@ -0,0 +1,54 @@
+import { describe, expect, it } from 'vitest';
+
+import { MemberConstants } from './constants.js';
+
+describe('Member constants', () => {
+  const invalidUsername = '<div>';
+  // prettier-ignore
+  const invalidUsername1 = 'user1234%66457657\'';
+  const invalidUsername2 = '  user\u{001A}';
+  const invalidUsername3 = '\u{008F}USER';
+  const validUsername = 'Johnny';
+  // prettier-ignore
+  const validUsername1 = 'Jéàn de l\'Avoiné';
+
+  it('username forbidden characters regex', () => {
+    expect(invalidUsername).toMatch(
+      MemberConstants.USERNAME_FORBIDDEN_CHARS_REGEX,
+    );
+    expect(invalidUsername1).toMatch(
+      MemberConstants.USERNAME_FORBIDDEN_CHARS_REGEX,
+    );
+    expect(invalidUsername2).toMatch(
+      MemberConstants.USERNAME_FORBIDDEN_CHARS_REGEX,
+    );
+    expect(invalidUsername3).toMatch(
+      MemberConstants.USERNAME_FORBIDDEN_CHARS_REGEX,
+    );
+
+    expect(
+      validUsername.match(MemberConstants.USERNAME_FORBIDDEN_CHARS_REGEX),
+    ).toBeFalsy();
```

Comment:
I'm not sure, but this should work too:

```suggestion
    expect(validUsername).not.toMatch(
      MemberConstants.USERNAME_FORBIDDEN_CHARS_REGEX,
    );
```

Reply:
Indeed !